### DISCUSSION TITLE
關於 use_preset_vocabulary 和 vocabulary 的設定

### DISCUSSION THREAD

**Discussion by markwu** [rime/home#1769] [General]
這邊有一個問題想請教，在方案製作的說明中說  vocabulary 是引入自定的詞庫設定，假設我有一個自定的詞庫 `essay_chewing.txt`，只包含 (詞, 詞頻) 就跟 `essay.txt` 一樣，我的字典檔設定如下

**mark_bopomo.dict.yaml**

```
name: mark_bopomo
version: "20201108"
sort: by_weight
vocabulary: essay_chewing
import_tables:
  - dict_tw/terra_pinyin
```
我透過 `vocabulary` 去引入詞庫，並且移除了 `use_preset_vocabulary: true` 的設定。

這時候，`terra_pinyin.dict.yaml` 中 還是保留了 `use_preset_vocabulary:true`。

請問這時候 `terra_pinyin.dict.yaml` 中的  `use_preset_vocabulary:true` 是引入八股文的詞庫和詞頻還是 `essay_chewing.txt`的詞庫和詞頻？

所以 `vocabulary` 真正的用途到底是什麼？

**Comment by lotem** [comment]
正在構建的詞典文件頭部的 YAML 配置有效.
`import_tables` 只導入其他文件中的碼表部分.

**Reply by markwu** [reply]
瞭解了，謝謝你！
